src/charge_point.cpp

```diff
@@ -100,8 +100,42 @@ int main(int argc, char* argv[]) {
         fs::create_directories(cso_path);
     }
 
+    const fs::path csms_path = "/tmp/client/csms";
+    if (!fs::exists(csms_path)) {
+        fs::create_directories(csms_path);
+    }
+
+    const fs::path ca_v2g_path = "/tmp/certs/ca/v2g/";
+    if (!fs::exists(ca_v2g_path)) {
+        fs::create_directories(ca_v2g_path);
+    }
+
+    const fs::path ca_mo_path = "/tmp/certs/ca/mo/";
+    if (!fs::exists(ca_mo_path)) {
+        fs::create_directories(ca_mo_path);
+    }
+
+    if (!fs::is_regular_file("/tmp/certs/ca/v2g/V2G_CA_BUNDLE.pem")) {
+        std::ofstream("/tmp/certs/ca/v2g/V2G_CA_BUNDLE.pem");
```

Comment:
See also the conditional use of `boost::filesystem::ofstream` in libevse-security:

https://github.com/EVerest/libevse-security/blob/c6e4f7e22363cd43bf3ee4c3930356dae2dafd43/include/support_older_cpp_versions.hpp#L13-L19

https://github.com/EVerest/libevse-security/blob/fb097c945812a738def096e44a66a895b2c052a0/include/evse_utilities.hpp#L54

(I'm not sure whether that's important to use, and if it doesn't just map to `std::ofstream` anyway.)